pep-0696.rst

```diff
@@ -234,6 +240,8 @@ or be bound in an outer class or function scope.
    class slice(Generic[StartT, StopT, StepT]): ...
                      # ^^^^^^ Invalid: ordering does not allow StopT to be bound
 
+Currently using a ``TypeVarLike`` from an outer scope as a default is not supported.
```

Comment:
Same as comment above. I recommend removing the word "Currently" because it's confusing.